### ISSUE TITLE
Make Version class constructor constant one

### ISSUE THREAD

**Issue by ycherniavskyi** [dartninja/version#19]
It will be great to make `Version` class constructor a constant one, because this will allow using the const `Version` instance as a default value of an optional parameter.

Also, I see such change is fully backward compatible.

**Comment by ycherniavskyi** [comment]
For sure, I could make a PR if it is necessary. But this improvement is only about adding `const` before the `Version` constructor.

**Comment by ycherniavskyi** [comment]
Hmm, it seems everything not such easy as I expected :)
```
lib/version.dart:38:3: Error: A const constructor can't have a body.
```

**Comment by sanmadjack** [comment]
I agree, making it constant would be great, and I wanted it to be initially. Unfortunately the preRelease arg being a List basically means that it cannot be as dart currently is. I just gave it another shot and it's a no-go. I'm open to an implementation if someone has an idea.
